Java/JDI/jdi-uitest-web/src/main/java/com/epam/jdi/uitests/web/selenium/elements/complex/Selector.java

```diff
@@ -47,6 +53,27 @@ public Selector(By optionsNamesLocatorTemplate, By allOptionsNamesLocator) {
         super(optionsNamesLocatorTemplate, allOptionsNamesLocator);
     }
 
+    public static void setUp(BaseElement el, Field field) {
+        if (!fieldHasAnnotation(field, JSelector.class, ISelector.class))
+            return;
```

Comment:
Please surround with curly braces. And try to avoid if-else (and similar code structures) statements without using curly.